cmd/config/internal/commands/cmdset_test.go

```diff
@@ -453,6 +453,58 @@ spec:
  `,
 			errMsg: `list in body must be of type integer: "string"
 list in body must be of type integer: "boolean"
+list in body should have at most 2 items`,
+		},
+
+		{
+			name: "validate openAPI list values with flag",
```

Comment:
Add another test that checks the "value begins with a dash" case -- e.g. `--value --hello world`.  Also test this for list values.

Reply:
Also add a test for if the value is specified as a flag AND and arg -- this should be an error

Reply:
Done.